ppanggolin/annotate/annotate.py

```diff
@@ -1034,12 +1034,10 @@ def check_chevrons_in_start_and_stop(
                             dbxref_metadata
                         )
 
-                    if fields_gff[gff_seqname] in circular_contigs or (
+                    if contig is not None and (
```

Comment:
Did you write this change because you met the case of having a "region" sequence that was circular without having the object be created?
After re-reading the code as it is now, do you think it is possible to be in the following case:
- "##sequence-region" is not there
- we create a Contig object when meeting the first CDS
- The "region" attributes are not properly associated to said contig object, for example if the circularity is only given through the "region" attributes we won't get it ?

Reply:
yes this scenario is possible. I modified this part to keep the info when contig is not iniitialized yet by using the circular_contigs list.